src/orchestratord/src/controller/materialize.rs

```diff
@@ -85,10 +85,6 @@ pub struct Config {
     pub clusterd_node_selector: Vec<KeyValueArg<String, String>>,
     pub clusterd_affinity: Option<Affinity>,
     pub clusterd_tolerations: Option<Vec<Toleration>>,
-    pub console_node_selector: Vec<KeyValueArg<String, String>>,
-    pub console_affinity: Option<Affinity>,
-    pub console_tolerations: Option<Vec<Toleration>>,
```

Comment:
Should we have fields for these in the ConsoleSpec?

Reply:
possibly - i was trying to keep this pr down to just moving code around, but it's something we may want to revisit in a future pr